flowershop/models/stock_warehouse.py

```diff
@@ -0,0 +1,105 @@
+from odoo import fields,models
+from odoo.exceptions import UserError,ValidationError
+import requests
+from datetime import datetime,time
+
+
+
+class Stockwarehouse(models.Model):
+    
+    _inherit = 'stock.warehouse'
+ 
+
+    weather_ids = fields.One2many('stock.warehouse.weather',
+    'weather_rec',
```

Comment:
```suggestion
    'warehouse_id',
```

TIP: a better name -> more descriptive and follows the naming convention